website/docs/search.md

```diff
@@ -6,18 +6,23 @@ keywords:
   - search
 ---
 
-Docusaurus' own `@docusaurus/preset-classic` supports a search integration.
+There are a few options you can use to add search to your website:
 
-There are two main options, you can use [Algolia DocSearch](https://docsearch.algolia.com) or bring in your own `SearchBar` component.
+- [Algolia DocSearch](#using-algolia-docsearch)
+- [Typesense DocSearch](#using-typesense-docsearch)
+- [Local Search](#using-local-search)
+- [Your own `SearchBar` component](#using-your-own-search)
```

Comment:
I suggest adding a note that says only the Algolia plugin is official, and the other solutions are maintained/supported by the community, otherwise, we may have users opening Typesense issues in this repo

Reply:
Sounds good, thank you for adding this!